globals.go

```diff
@@ -956,11 +956,10 @@ var (
 		HostStarting,
 	}
 
-	// Hosts in "initializing" status aren't actually running yet:
+	// Hosts in "initializing" and "building" status aren't actually running yet:
```

Comment:
I'm not able to open the ticket because of the Jira outage, but is the context for this the idea of omitting building hosts from the task queue page? Can that query instead just use the running status? (That's the _only_ status that's able to run tasks in this list, so its the only status we should consider). The comment as you've edited it here is false; "Building" hosts are running hosts.